bftengine/src/bftengine/messages/PrePrepareMsg.hpp

```diff
@@ -155,7 +155,7 @@ class RequestsIterator {
 
 template <>
 inline MsgSize maxMessageSize<PrePrepareMsg>() {
-  return ReplicaConfig::instance().getmaxExternalMessageSize() + MessageBase::SPAN_CONTEXT_MAX_SIZE;
+  return ReplicaConfig::instance().getmaxExternalMessageSize() - MessageBase::SPAN_CONTEXT_MAX_SIZE;
```

Comment:
looks good.
but why do we need to subtract SPAN_CONTEXT_MAX_SIZE? 
If SPAN_CONTEXT is added after the PPM is framed, then it looks fine and can you pls mention the same in the PR description? Thanks.

Reply:
Because if we exceeds the maxExternalMessagesSize, the messages will be dropped by the communication layer

Reply:
So spans has to be part of the message itself